### PR TITLE
As/hw2

### PR DESCRIPTION
**Pull request by MrTyoom**

### CODE REVIEW

**Comment by yana-tanch** [comment]
- Данные:

1. В train_dataset используются files 
train_dataset = DataSet(files, train_transform)

2. Очень большие размеры батчей 
 
- Обучение:

1. scheduler.step() до вычисления метрик


- Тестирование:
 
1.  Копируем, но не используем
x_cpu = x.cpu() 

2. На тестовом наборе не должны применяться аугментации

3. Очень большой размер батча

4. accuracy.update(y_pred.cpu(), y.cpu()) за пределами цикла 

5. Отсутствует model.eval()

**Comment by MrTyoom** [comment]
Исправлено

**Comment by yana-tanch** [comment]
Значение точности нужно улучшать, даже с полносвязной архитектурой можно выбить больше

**Comment by MrTyoom** [comment]
Последний результат, который получился 58.4%

**Comment by MrTyoom** [comment]
Исправлено

**Comment by yana-tanch** [comment]
Если не мудрить с аугментацией и оставить только T.RandomHorizontalFlip(p=0.5) на train, на тесте уже 60.5% получится :)

**Review by yana-tanch — APPROVED** [review]
